apps/arweave/src/ar_chunk_storage.erl

```diff
@@ -343,33 +371,69 @@ handle_info({chunk, {packed, Ref, ChunkArgs}},
 	case maps:get(Ref, Map, not_found) of
 		not_found ->
 			{noreply, State};
-		Offset ->
+		Args ->
 			State2 = State#state{ packing_map = maps:remove(Ref, Map) },
-			{Packing, Chunk, _, _, _} = ChunkArgs,
-			case ar_sync_record:delete(Offset, Offset - ?DATA_CHUNK_SIZE,
-					ar_data_sync, StoreID) of
+			{Packing, Chunk, Offset, _, ChunkSize} = ChunkArgs,
+			StartOffset = Offset - ?DATA_CHUNK_SIZE,
+			RemoveFromSyncRecordResult = ar_sync_record:delete(Offset,
```

Comment:
we remove here in expectation that we'll re-add the chunk to `ar_data_sync` record when handling the store_chunk` message?